apps/dashboard/app/models/user_configuration.rb

```diff
@@ -91,6 +91,13 @@ def public_url
     path.start_with?('/') ? Pathname.new(path) : Pathname.new('/public')
   end
 
+  # The file system path to the announcements
+  # @return [Pathname, Array<Pathname>] announcement path or paths
+  def announcement_path
+    paths = ENV['OOD_ANNOUNCEMENT_PATH'] || fetch(:announcement_path, ['/etc/ood/config/announcement.md', '/etc/ood/config/announcement.yml', '/etc/ood/config/announcements.d'])
```

Comment:
Can you use the same if/else block from the previous code. that feels more readable than a one liner || statement.

Reply:
Fixed